wg-meetings/2018-02-21.md

```diff
@@ -0,0 +1,90 @@
+# Node.js Foundation Diagnostics Working Group Meeting 2018-02-07 Minutes
+
+## Attendees
+ - @mike-kaufman
+ - @watson
+ - @qard (Stephen Belanger)
+ - @mmarchini
+ - @fishrock123 (Jeremiah Senkpiel)
+ - @mhdawson (Michael Dawson)
+
+
+## Links
+* **This Meeting**:
+  * Minutes:  https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2018-02-21.md
+  * Notes: https://docs.google.com/document/d/1ucTWrJuU7YA-PA2DQraqzIjp3mhf0cyx3tmgGiYw-RA/edit?usp=sharing
+  * You Tube: http://youtu.be/RyUR6Ec4BQs
+* **Previous Meeting**: 
+  * Minutes:   https://github.com/nodejs/diagnostics/blob/master/wg-meetings/2018-02-07.md 
+  * Notes: https://docs.google.com/document/d/1BFmXdY4X2v_LMSSifyvaf6LzMx6nuZx1pbXUJQUTINU/edit?usp=sharing
+  * You Tube:  http://youtu.be/8iu59kEIhzE
+
+## Agenda
+
+Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* Integrate C++ AsyncHooks Embedder API with native abstraction [#13254](https://github.com/nodejs/node/issues/13254)
+ - A PR to NAN is merged. Need to revisit when Ali is around.
+
+### nodejs/diagnostics
+
+* Diag WG meeting times? [#158](https://github.com/nodejs/diagnostics/issues/158)
+  - two proposed times in issue, 9 AM & 11 AM
+  - TSC conflict would occur once every 6 weeks with 9 AM slot
+  - moderation conflict would occur infrequently with 11 AM slot
+    - moderation meets once a month and is currently out-of-phase with Diag WG meetings
+  - **ACTIONS:** 
+    - [x] @mike-kaufman - update #158 with proposed 11AM time, people can chime in there with thumbs-up/thumbs-down.  
+
+* Deep dive meeting on CI testing to ensure tools compatibility [#156](https://github.com/nodejs/diagnostics/issues/156)
+  - probably too early to deep-dive CI testing
+  - believe @joyeecheung owned getting details on this after summit

```

Comment:
Is it referring to https://github.com/nodejs/diagnostics/issues/157 ?

Reply:
This was specifically referring to #156, but I think #157 is a pre-cursor to getting anything in CI.  Would be useful if someone could synthesize the different pieces here - i.e., propose in more detail what needs to happen & the order it needs to happen.